repository-data/application/src/main/resources/hcm-content/content/documents/administration/valuelists/lks/searchbankproviders.yaml

```diff
@@ -313,7 +314,8 @@
     /selection:listitem[76]:
       jcr:primaryType: selection:listitem
       selection:key: university_hospitals_birmingham_uhb_library_and_knowledge_service
-      selection:label: University Hospitals Birmingham (UHB) Library and Knowledge Service
```

Comment:
lets revert this so that it's on one line

Reply:
Done